Makefile

```diff
@@ -134,9 +138,7 @@ test: generate
 	ginkgo -skipPackage test/e2e/dcos,test/e2e/kubernetes -failFast -r .
 
 .PHONY: test-style
-test-style:
-	@scripts/validate-shell.sh
-	@scripts/validate-go.sh
+test-style: validate-go validate-shell validate-copyright-headers
```

Comment:
the pipeline (`pr-e2e.yaml`) calls these separately so we should update it as well

Reply:
https://github.com/Azure/aks-engine/blob/master/.pipelines/pr-e2e.yaml#L55

Reply:
I thought about doing that, but `make` will actually sort out the dependent targets so that `make validate-copyright-headers` isn't called twice. But you're right that it's cleaner, so I'll change it.